### ISSUE TITLE
OP with large number of tensor args fails under tracy build due to MSG size limit

### ISSUE THREAD

**Issue by csehydrogen** [tenstorrent/tt-metal#29519]
### Describe the bug

Under tracy build, json dump containing OP information are recorded as TracyMessage and later written to `tracy_ops_data.csv`. TracyMessage has a limit of 64K bytes, so OP with lots of tensor args (around 128) may exceed the limit.

AFAIK, only OP in tt-metal that accepts variable number of tensor args is `ttnn.concat`. I found `ttnn.concat` somehow hangs with 80 input tensors, thus cannot reproduce bug as-is. Thus, I made a reproducing example by emptying the concat implementation.

### Steps to reproduce the issue

1. Checkout `csehydrogen/tracy_msg_64k_limit` branch
2. Build with profiler enabled
3. Run the following script
```
import torch
import ttnn
device = ttnn.open_device(device_id=0)
a_torch = torch.rand(32, 32, dtype=torch.bfloat16)
a_ttnn = ttnn.from_torch(a_torch, layout=ttnn.TILE_LAYOUT, device=device)
n = 128
inputs = [a_ttnn for _ in range(n)]
b_ttnn = ttnn.concat(inputs)
```
```
python: /root/tt-metal/tt_metal/third_party/tracy/public/tracy/../client/TracyProfiler.hpp:394: static void tracy::Profiler::Message(const char *, size_t, int): Assertion `size < (std::numeric_limits<uint16_t>::max)()' failed.
```

### Expected behavior

The assert should not happen.

**Comment by ihnpark551** [comment]
Hi @zzigler-tt, this item has been newly created as P1 issue, which issue affects performance metric check on our dev work.
We made related [PR](https://github.com/tenstorrent/tt-metal/pull/29523) to solve this ticket, 
and it would be great if TT team can check ASAP!

**Comment by zzigler-tt** [comment]
@ihnpark551 Sure thing, the team is currently working through PR review/approval.

**Comment by csehydrogen** [comment]
Close as #29523 merged.
